src/ibrowse_http_client.erl

```diff
@@ -463,9 +463,14 @@ accumulate_response(Data, #state{reply_buffer      = RepBuf,
             State#state{reply_buffer = RepBuf_1}
     end.
 
+strictly_monotonic_timestamp() ->
+   {A, B, _} = erlang:timestamp(),
+   Unique = erlang:unique_integer([positive, monotonic]),
+   {A, B, Unique}.
```

Comment:
To keep the microseconds in the format, in case the VM is restarted within one second, we could make the A be the unix seconds, C be microseconds then the unique value.